mne/io/base.py

```diff
@@ -1450,6 +1453,48 @@ def save(self, fname, picks=None, tmin=0, tmax=None, buffer_size_sec=None,
                    start, stop, buffer_size, projector, drop_small_buffer,
                    split_size, split_naming, 0, None, overwrite)
 
+    @verbose
+    def export(self, fname, fmt='auto', verbose=None):
+        """Export Raw to external formats.
+
+        Supported formats: EEGLAB (set, uses :mod:`eeglabio`)
+        %(export_warning)s
+        %(export_params_base)s
```

Comment:
I think it would make sense to include the supported parameters right there. As in,

```
fmt : 'set' | 'abc' | 'def'

Reply:
I see, but we should also keep the "supported formats" section to include more details, right?